zilliqa/Cargo.toml

```diff
@@ -33,10 +33,11 @@ bs58 = "0.5.0"
 bytes = "1.5.0"
 chrono = "0.4"
 clap = { version = "4.4.13", features = ["derive"] }
-crypto-bigint = "=0.5.5"
+crypto-bigint = "=0.5.0"
```

Comment:
by the way, version 0.5.5 of crypto-bigint breaks my editor which is why I keep reverting it. Since it is a minor version it would be nice if it could be frozen for now

Reply:
It is referenced here: https://youtrack.jetbrains.com/issue/RUST-11841/Can-not-run-nested-type-inference-types-and-highlighting-in-the-code-are-not-displayed

The behaviour in my editor is that it randomly corrupts hotkeys, highlighting comes on only sometimes, and it randomly deletes when you want to insert text. So, it makes it highly unusable which is quite surprising for a random library.